<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyGeneration</class>
 <widget class="QMainWindow" name="KeyGeneration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Keys</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="goButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectFilepathButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Filepath For Keys</string>
    </property>
   </widget>
   <widget class="QComboBox" name="KeySizeComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>128</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>256</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>512</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1024</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2048</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4096</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="KeySizeLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Key Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="ImageLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
